src/main/java/org/alliancegenome/curation_api/base/entity/AuditedObject.java

```diff
@@ -47,11 +48,13 @@ public class AuditedObject extends BaseEntity {
     @FullTextField(analyzer = "autocompleteAnalyzer", searchAnalyzer = "autocompleteSearchAnalyzer", valueBridge = @ValueBridgeRef(type = OffsetDateTimeValueBridge.class))
     @KeywordField(name = "dateCreated_keyword", sortable = Sortable.YES, searchable = Searchable.YES, aggregable = Aggregable.YES, valueBridge = @ValueBridgeRef(type = OffsetDateTimeValueBridge.class))
     @JsonView({View.FieldsOnly.class})
+    @CreationTimestamp
     private OffsetDateTime dateCreated;
     
     @FullTextField(analyzer = "autocompleteAnalyzer", searchAnalyzer = "autocompleteSearchAnalyzer", valueBridge = @ValueBridgeRef(type = OffsetDateTimeValueBridge.class))
     @KeywordField(name = "dateUpdated_keyword", sortable = Sortable.YES, searchable = Searchable.YES, aggregable = Aggregable.YES, valueBridge = @ValueBridgeRef(type = OffsetDateTimeValueBridge.class))
     @JsonView(View.FieldsOnly.class)
+    @UpdateTimestamp
```

Comment:
Again, not sure we want this due to overwriting of submitted dateUpdated.

Reply:
if we call an update on a object doesn't matter source I would think we would want this one to fire.

Reply:
I think it would also overwrite dateUpdated values coming in from the DQM submissions - I would think that we would want to keep any submitted values rather than have the date of loading

Reply:
I would think we would want to indicate that we updated the record when it was loaded into the db? Also the @ColumnDefault wont work on this because saving will just take the incoming value and keep it. So to me anytime we touch a record we should update its timestamp and its modified by.

Reply:
For instance with MGI we set the updated_by column to the "load" process so that we know it was "last touched" by the load vs a curation. Plus in our case here we have the AUD tables so we can see the changes.

Reply:
Fair enough, if we want the date of loading then this saves a lot of hassle.  We should remove the code where this is explicitly set (I think possibly only the AuditedObjectValidator).

Reply:
Although I do still wonder whether the date of loading is that useful for a curator vs. when the annotation was actually last changed